### ISSUE TITLE
Doesn't install without C extensions.

### ISSUE THREAD

**Issue by travishaynes** [postmodern/digest-crc#22]
The README says the C extensions are optional, and that if it fails for _whatever_ reason, it will fallback to a pure-Ruby implementation. However, I couldn't get it to install without the C extensions. To reproduce run:

```bash
docker run -it ruby:2.7.1-alpine gem install digest-crc
```

That container doesn't have any build tools installed by default, so it can't install the C extensions. It fails to install with compiler errors.

**Comment by postmodern** [comment]
Ah it appears that rake's `ruby(...)` and `sh(...)` methods raise a RuntimeError when the command fails. e68b53cc826bc779391726f0e97815f3ff6ca879 should fix it. Tested by building the gem locally with `gem build ./digest-crc.gemspec` and built a custom docker image using:

```Dockerfile
FROM ruby:2.5.0-alpine

RUN apk add --no-cache bash

COPY ./digest-crc-0.6.2.gem .
```

```
$ docker build -t digest-crc .
$ docker run -it digest-crc gem install ./digest-crc-0.6.2.gem
```

Could you test that on your end before I close this and release 0.6.2?

**Comment by postmodern** [comment]
Should be fixed in 0.6.2. Added a note to the [README](https://github.com/postmodern/digest-crc#install) and added [Integration Testing](https://github.com/postmodern/digest-crc/tree/master/spec/integration) where we install a freshly built `digest-crc.gem` into a variety of docker images.
